src/main/generated/resources/assets/minestuck/lang/en_us.json

```diff
@@ -53,46 +53,47 @@
+  "block.minestuck.attached_strawberry_stem":        "Strawberry Stem",
+  "block.minestuck.black_chess_dirt":                "Black Chess Tile",
+  "block.minestuck.black_stone":                     "Black Stone",
+  "block.minestuck.blender":                         "Blender",
+  "block.minestuck.blood":                           "Blood",
+  "block.minestuck.blood_aspect_leaves":             "Blood Leaves",
+  "block.minestuck.blood_aspect_log":                "Blood Log",
+  "block.minestuck.blood_aspect_planks":             "Blood Planks",
+  "block.minestuck.blood_aspect_sapling":            "Blood Sapling",
+  "block.minestuck.blood_aspect_sapling.tooltip":    "Not yet implemented",
+    "block.minestuck.blooming_cactus":               "Blooming Cactus",
+    "block.minestuck.blue_cake":                     "Blue Cake",
+    "block.minestuck.blue_dirt":                     "Blue Dirt",
+    "block.minestuck.brain_juice":                   "Brain Juice",
+    "block.minestuck.breath_aspect_leaves":          "Breath Leaves",
+    "block.minestuck.breath_aspect_log":             "Breath Log",
+    "block.minestuck.breath_aspect_planks":          "Breath Planks",
+    "block.minestuck.breath_aspect_sapling":         "Breath Sapling",
+    "block.minestuck.breath_aspect_sapling.tooltip": "Not yet implemented",
+    "block.minestuck.bright_dense_cloud":            "Bright Dense Cloud",
+    "block.minestuck.cassette_player_default":       "Cassette Player",
+    "block.minestuck.cast_iron":                     "Cast-Iron",
+    "block.minestuck.cast_iron_stairs":              "Cast-Iron Stairs",
+    "block.minestuck.chalk":                         "Chalk",
+    "block.minestuck.chalk_brick_slab":              "Chalk Brick Slab",
+    "block.minestuck.chalk_brick_stairs":            "Chalk Brick Stairs",
+    "block.minestuck.chalk_bricks":                  "Chalk Bricks",
+    "block.minestuck.chalk_slab":                    "Chalk Slab",
+    "block.minestuck.chalk_stairs":                  "Chalk Stairs",
+    "block.minestuck.chessboard":                    "Chessboard",
+    "block.minestuck.chiseled_cast_iron":            "Chiseled Cast-Iron",
+  "block.minestuck.chiseled_chalk_bricks":           "Chiseled Chalk Bricks",
+  "block.minestuck.chiseled_coarse_stone":           "Chiseled Coarse Stone",
+  "block.minestuck.chiseled_frost_bricks":           "Chiseled Frost Bricks",
+  "block.minestuck.chiseled_pink_stone_bricks":      "Chiseled Pink Stone Bricks",
+  "block.minestuck.coagulated_blood":                "Coagulated Blood",
+  "block.minestuck.coarse_end_stone":                "Coarse End Stone",
+  "block.minestuck.coarse_stone":                    "Coarse Stone",
+  "block.minestuck.coarse_stone_stairs":             "Coarse Stone Stairs",
+  "block.minestuck.cobblestone_cruxite_ore":         "Cruxite Ore",
+  "block.minestuck.cobblestone_uranium_ore":         "Uranium Ore",
```

Comment:
So uh, did you do some formatting here on accident?
I'd suggest rerunning genData to fix it